rules/watch-build.mjs

```diff
@@ -1,13 +1,12 @@
 import fs from 'fs';
```

Comment:
This file is not used anymore, right? Let's remove it? `npm run watch` doesn't rely on it